index.js

```diff
@@ -12,15 +12,20 @@ module.exports = {
     this._super.included.apply(this, arguments);
 
     var hostApp = this._findApp(app);
-    this._regeneratorAlreadyIncluded =
+
+    var babelOptions = (hostApp.options && hostApp.options.babel) || {};
+    var emberCLIBabelOptions = (hostApp.options && hostApp.options['ember-cli-babel']) || {};
+
+    var babelInstance = this.addons.filter(function(addon) { return addon.name === 'ember-cli-babel'; })[0];
```

Comment:
ya, but I kept es5 compat to follow along with the rest of the things here.

In theory we'd have to do a major bump to do es6y things, and I'd like to get this out in 1.x...